pkgs/top-level/all-packages.nix

```
@@ -18759,6 +18759,8 @@ with pkgs;
 
   sawjap = callPackage ../development/tools/java/sawjap { };
 
+  sbomnix = callPackage ../tools/security/sbomnix { };
```

```suggestion
  sbomnix = python3.pkgs.callPackage ../tools/security/sbomnix { };
```

This seems to be the preferred method for python applications

---

Fixed in 0f1b2e7747bb9dea653ca76d107028bc25450b22